src/GPS/GPSProvider.cc

```diff
@@ -202,7 +203,7 @@ int GPSProvider::callback(GPSCallbackType type, void *data1, int data2)
         }
         case GPSCallbackType::writeDeviceData:
             if (_serial->write((char*) data1, data2) >= 0) {
-                if (_serial->waitForBytesWritten(-1))
+                //if (_serial->waitForBytesWritten(-1))
```

Comment:
@bkueng It looks like this may be the problem on android.  I can get RTK in a working state if I comment out this check.

Reply:
Hmm, I don't remember why I added this. It might not be required at all.

Reply:
Is waitForBytesWritten simply not support by the android serial port code? Might not be.